src/containers/TopicPage/TopicPage.jsx

```diff
@@ -189,9 +189,10 @@ class TopicPage extends Component {
                 topicPath={topicPath}
               />
               <Resources
+                title={article.title}
```

Comment:
This fails will fail if article is null (see https://ndla-frontend-pr-199.ndla.sh/subjects/subject:9/topic:1:182163/topic:1:156119 vs https://ndla-frontend.test.api.ndla.no/subjects/subject:9/topic:1:182163/topic:1:156119).  Use topic.name instead.